benchmark/events_histogram_op_benchmark.cpp

```diff
@@ -12,6 +12,7 @@
 
 using namespace scipp;
 
+/*
```

Comment:
I assume there is a reason but why is this whole file commented out?

Reply:
Yes, burnout/lazyness. I am adding this to the to-do in #1363, thanks for pointing it out.